docs/build/optimisation.md

```diff
@@ -0,0 +1,36 @@
+# Project Optimisation
+
+Performance is a crucial factor in each project. So, how to optimise your Suproject to speed it up?
+Fortunately, there are several things you could do to improve it. 
+
+### Common Issues and Top Suggestions
+
+- Avoid using `blockHandlers` where possible. Using block handlers slows your project down as they can be executed with each and every block. Use them only if you need to and [consider adjusting project architecture](#rethink-project-architecture).
+- Always use a [dictionary](../tutorials_examples/dictionary.html#how-does-a-subquery-dictionary-work) (we can help create one for your new network). 
+- Try to use filter conditions to reduce the response size. Create filters as specific as possible to avoid querying unnecessary data. 
+
+### Other Improvements 
+
+ Here is the list of further suggestions:
+
+- Query only necessary fields. 
```

Comment:
Not sure what this one means?

Reply:
Oh, from the query service

Perhaps we need to split up indexing speed advice from query speed advice